packages/instant/src/components/payment_method_dropdown.tsx

```diff
@@ -0,0 +1,49 @@
+import { BigNumber } from '@0x/utils';
+import copy from 'copy-to-clipboard';
+import * as React from 'react';
+
+import { Network } from '../types';
+import { etherscanUtil } from '../util/etherscan';
+import { format } from '../util/format';
+
+import { Dropdown, DropdownItemConfig } from './ui/dropdown';
+
+export interface PaymentMethodDropdownProps {
+    selectedEthAddress: string;
+    addressEthBaseAmount: BigNumber;
+    network: Network;
+}
+
+export class PaymentMethodDropdown extends React.Component<PaymentMethodDropdownProps> {
+    public static defaultProps = {
```

Comment:
Even though this PR is just adding this for display purposes, it feels dangerous to have these defaultProps here -- let's make sure we remove them